### PR TITLE
Fix UG Related Bugs

### PR DESCRIPTION
**Pull request by seanlaiys**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -414,7 +421,7 @@ Format: `mark_attendance tn/TUTORIAL_NAME [id/STUDENT_ID] wk/WEEK`
 
 Examples:
 
-* `mark_attendance tn/T04 id/e0123456 wk/1`
+* `mark_attendance tn/T04 n/e0123456 wk/1`
```
Comment (zixin448) [inline docs/UserGuide.md:424]: is this supposed to be id/?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -485,21 +492,27 @@ _Details coming soon ..._
...
 | **Add**               | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665  t/friend t/colleague`               |
-| **Add Class**         | `add_class tn/TUTORIAL_NAME v/VENUE d/DAY tm/TIME` <br> e.g., `add_class c/T04 v/LT13 d/Monday t/13:00 wk/13`                                                                               |
-| **Add Student**       | `add_student n/NAME id/STUDENT_ID tn/TUTORIAL_NAME` <br> e.g., `add_student n/Amy Tan id/e0123456 tn/T13`                                                                                    |
-| **Add Assessment**    | `add_assessment as/ASSESSMENT_NAME w/WEIGHTAGE f/FULL MARK` <br> e.g., `add_assessment as/Attendance w/5 f/10`                                                                            |
+| **Add Class**         | `add_class tn/TUTORIAL_NAME v/VENUE d/DAY tm/TIME` <br> e.g., `add_class c/T04 v/LT13 d/Monday t/13:00 wk/13`                                                                        |
```
Comment (zixin448) [inline docs/UserGuide.md:495]: can add wk/WEEK to this part!

**Review by zixin448 — COMMENTED** [review]
looks good! just some small things to take note

File: docs/UserGuide.md [hunk 3]
```diff
@@ -356,9 +363,9 @@ Removes a student from a given class, replaces the `Student` contact with a `Per
 Format: `remove_student INDEX tn/TUTORIAL_NAME` or `remove_student id/STUDENT_ID tn/TUTORIAL_NAME`.
 
 * `list_student` has to be called before `remove_student` as this is the list referred to by the `remove_student` command
-* Removes the student with the specified `INDEX` or `STUDENT_ID` from the class with specified `TUTORIAL_NAME`.
+* Removes the student with the specified `INDEX` or `NAME` from the class with specified `TUTORIAL_NAME`.
```
Comment (garethkoh) [inline docs/UserGuide.md]:
```suggestion
* Removes the student with the specified `INDEX` or `STUDENT_ID` from the class with specified `TUTORIAL_NAME`.
```

File: docs/UserGuide.md (hunk 1 above)
Comment (garethkoh) [inline docs/UserGuide.md]:
```suggestion
* `mark_attendance tn/T04 id/e0123456 wk/1`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -485,21 +492,27 @@ _Details coming soon ..._
...
 | **Clear**             | `clear`                                                                                                                                                                              |
 | **Delete**            | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                  |
 | **Delete Class**      | `delete_class INDEX [tn/TUTORIAL_NAME]` <br> e.g., `delete_class 1 [tn/G04]`                                                                                                         |
-| **Remove Student**    | `remove_student INDEX tn/TUTORIAL_NAME` <br> e.g. `remove_student 1 tn/G04`                                                                                                     |
+| **Remove Student**    | `remove_student INDEX tn/TUTORIAL_NAME` <br> e.g. `remove_student 1 tn/G04`                                                                                                          |
```
Comment (garethkoh) [inline docs/UserGuide.md:501]: can add `remove_student id/STUDENT_ID tn/TUTORIAL_NAME` <br> e.g. `remove_student id/e0123456 tn/G04`

**Review by garethkoh — CHANGES_REQUESTED** [review]

**Pull request merged**